etc/crontab.xml

```diff
@@ -4,7 +4,7 @@
 		<job name="yotpo_yotpo_orders_sync" instance="Yotpo\Yotpo\Model\Jobs\OrdersSync" method="execute">
             <schedule>* * * * *</schedule>
         </job>
-		<job name="yotpo_yotpo_update_metadata" instance="Yotpo\Yotpo\Model\Jobs\UpdateMetadata" method="execute">
+		<job name="yotpo_yotpo_update_metadata" instance="Yotpo\Yotpo\Model\Jobs\UpdateMetadataJob" method="execute">
```

Comment:
This change is unlikely what you want. This is not referencing `Yotpo\Yotpo\Cron\UpdateMetadataJob` but `Yotpo\Yotpo\Model\Jobs\UpdateMetadata`. `Yotpo\Yotpo\Model\Jobs\UpdateMetadataJob` does not exist.

Reply:
@fredden, Thanks for noticing.
I actually solved it in a different way on:
https://github.com/YotpoLtd/magento2-module-yotpo-reviews/pull/247